### PR TITLE
Solution method Sort

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Review by senile-pitcher — APPROVED** [review]
👍

File: src/arrayMethodSort.js [hunk 1]
```diff
@@ -4,9 +4,30 @@
...
+    for (let i = 0; i < this.length; i++) {
+      for (let j = 0; j < this.length; j++) {
+        if (compareFunction(this[j], this[i]) > 0) {
+          const temporaryValue = this[i];
+
+          this[i] = this[j];
+          this[j] = temporaryValue;
+        }
+      }
+    }
```
Comment (Olexandr6) [inline src/arrayMethodSort.js:27]: How many iterations will be if the array is already sorted?

File: src/arrayMethodSort.js [hunk 2]
```diff
@@ -4,9 +4,30 @@
...
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  function compareString(a, b) {
+    if (String(a) > String(b)) {
+      return 1;
+    } else if (String(a) < String(b)) {
+      return -1;
+    } else {
+      return 0;
+    }
```
Comment (Olexandr6) [inline src/arrayMethodSort.js:14]: You used String() constructor 4 times, let's use only 2 times

**Review by G-YaSha — CHANGES_REQUESTED** [review]
Your solution is good👍, but can be improved

File: src/arrayMethodSort.js (hunk 1 above)
Comment (Olexandr6) [reply src/arrayMethodSort.js:27]: Read about a flag in bubble sort

**Review by jstmpelowycz — APPROVED** [review]

**Pull request closed without merging**